src/test/java/org/takes/rq/RqFormTest.java

```diff
@@ -86,4 +86,49 @@ public void sameInstance() throws IOException {
             Matchers.is(Boolean.TRUE)
         );
     }
+
+    /**
+     * RqForm.Fake can create fake forms with parameters list.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void createsFormRequestWithParams() throws IOException {
```

Comment:
@erimerturk Should declare `throws Exception`, as per the recommendation [in this post](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names).